docs/developers/0_preparation.md

```diff
@@ -0,0 +1,22 @@
+# 💼 Preparing a development environment
+In order to compile ReVanced Manager, certain requirements must be met.
+
+## 📝 Prerequisites
+* Kotlin IDE such as [Android Studio](https://developer.android.com/studio)
+* Knowledge of [Android](https://android.com) App development, [Kotlin](https://kotlinlang.org/) language.
+* At least JDK 17 of any vendors
+
+## 🏃 Prepare the environment
+1. Clone the repository
+   ```sh
+   git clone https://github.com/ReVanced/revanced-manager.git && cd revanced-manager
+   ```
+2. Build the APK
+   ```sh
+   gradlew assembleRelease -Psign
```

Comment:
Nope nevermind, we still uses GitHub package for RVMC 😭 @Axelen123 plz fix

Reply:
Umm, no? RVMC switched to jitpack as well.

Reply:
> @Axelen123 Umm, no? RVMC switched to jitpack as well.

😭 oh my god, I'm idiot, I look `docs/readme` branch, not `compose-dev`